src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -772,8 +772,8 @@ public sealed partial class LoaderOptimizationAttribute : System.Attribute
     }
     public static partial class Math
     {
-        public const double E = 2.7182818284590451;
-        public const double PI = 3.1415926535897931;
+        public const double E = 2.718281828459045;
```

Comment:
Is it OK To change the precision of these constants?

Reply:
Santi already checked with me. The underlying bits arent changing.